lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p1-identity.yaml

```diff
@@ -128,20 +128,8 @@ states:
         checkIfDisabled:
           dcmaw:
             targetState: NINO_START_PAGE
-            checkIfDisabled:
-              hmrcKbv:
-                targetState: F2F_START_PAGE
-                checkIfDisabled:
-                  f2f:
-                    targetState: PYI_ESCAPE
       end:
         targetState: NINO_START_PAGE
-        checkIfDisabled:
-          hmrcKbv:
-            targetState: F2F_START_PAGE
-            checkIfDisabled:
-              f2f:
-                targetState: PYI_ESCAPE
 
```

Comment:
Think we still want f2f disabled check (and above) ?

Reply:
Would f2f being disabled affect the nino happy path? We have the f2f disabled check on the NINO_START_PAGE on the "end" event

Reply:
Ah yes of course